packages/grid/_modules_/grid/models/api/gridEditingApi.ts

```diff
@@ -40,6 +40,13 @@ export interface GridEditingSharedApi {
     params: GridEditCellValueParams,
     event?: MuiBaseEvent,
   ) => Promise<boolean> | void;
+  /**
+   * @ignore - do not document.
+   */
+  unstable_runPendingEditCellValueChangeDebounce: (
```

Comment:
Could you document the params even if it is not on the doc ?
Because the `maybeField` is kind of cryptic to me right now. I suppose that when there is no field it because we update the whole row.